scripts/testnet/sovereignBridge/config/common.snippets.sh

```diff
@@ -1,55 +1,59 @@
-deployMainChainContractsAndSetupObserver() {
-    deployEsdtSafeContract
-
-    deployFeeMarketContract
-
-    setFeeMarketAddress
-
-    disableFeeMarketContract
-
-    unpauseEsdtSafeContract
-
-    setGenesisContract
-
-    updateSovereignConfig
-
-    prepareObserver
+checkWalletBalanceOnMainChain() {
+    local BALANCE=$(mxpy account get --address ${WALLET_ADDRESS} --proxy ${PROXY} --balance)
+    if [ "$BALANCE" == "0" ]; then
+        echo -e "Your wallet balance is zero on main chain"
+        return 1
+    fi
+    return 0
 }
 
-sovereignDeploy() {
-    updateNotifierNotarizationRound
-
-    ../config.sh
-
-    deployMultisigVerifierContract
-
-    setEsdtSafeAddress
-
-    updateAndStartBridgeService
-
-    ../sovereignStart.sh
-
-    deployObserver
-
-    setMultisigAddress
-
-    setSovereignBridgeAddress
-
-    getFundsInAddressSovereign
+getFundsInAddressSovereign() {
+    echo "Getting funds in wallet on sovereign chain..."
+
+    local OUTFILE="${OUTFILE_PATH}/get-funds-sovereign.interaction.json"
+    mxpy tx new \
+        --pem="~/MultiversX/testnet/node/config/walletKey.pem" \
+        --pem-index 0 \
+        --proxy=${PROXY_SOVEREIGN} \
+        --chain=${CHAIN_ID_SOVEREIGN} \
+        --receiver=${WALLET_ADDRESS} \
+        --value=200000000000000000000 \
+        --gas-limit=50000 \
+        --outfile=${OUTFILE} \
+        --recall-nonce \
+        --send
+
+    sleep 6
+}
 
-    setFeeMarketAddressSovereign
+gitPullAllChanges()
+{
+    pushd .
 
-    disableFeeMarketContractSovereign
+    # Traverse up to the parent directory of "mx-chain-go"
+    while [[ ! -d "mx-chain-go" ]]; do
```

Comment:
We should Have I think an extra exit condition here,
If you call it from another directory, this might act weird.

Reply:
fixed